tests/foreman/ui/test_host.py

```diff
@@ -1849,6 +1880,37 @@ def test_all_hosts_bulk_delete(target_sat, function_org, function_location, new_
         assert session.all_hosts.bulk_delete_all()
 
 
+@pytest.mark.tier2
+def test_multi_cv_read(target_sat, rhel_contenthost, function_org, new_host_ui):
```

Comment:
not sure if this was meant to be included, the description and prt run don't mention it

Reply:
You are right, this is indeed cruft from some additional work. I'll clean this up .